### DISCUSSION TITLE
SnipeIT and Google Sheets

### DISCUSSION THREAD

**Discussion by brianhoganm** [snipe/snipe-it#12939] [General]
Hello,

I was wondering if anyone has a way to pull SnipeIT assets into Google Sheets automatically. I am looking to have a Google Sheet with assets checked out to users from SnipeIT. Looking to pull Asset Barcode Number, Name of the person, asset model, serial, etc. I am **not** looking to make any changes to the Google Sheets or update it back to SnipeIT. Thanks!

**Comment by snipe** [comment]
Probably something like this: https://snipe-it.readme.io/docs/connecting-to-google-docs

**Reply by brianhoganm** [reply]
Hi Snipe. I tried following those directions and I was not able to get it to work. From my understanding, it only outputs/fetches the URL's of the locations right? I am looking for a way to simply display the assets in Google Sheets that are assigned to my users very similar to how it displays in the SnipeIT table. This way, I can show my teachers of what Chromebooks they have in their classrooms assigned to students. If there was a way to pull all the fileds than I can publish my Google Sheets. 

 I am looking to do this since Snipe-IT does not have the ability to allow users who are managers of their locations to view devices. 

I also tried to connect my SQL database to Google Sheets to see if I can pull those fileds but no success either. 
Thanks!
